source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -53,6 +53,102 @@ Once you have finished this step, you should have:
 
   * Kibana - <https://kibana.{hostname}>
   * OpenShift portal - <https://openshift.{hostname}>
+  
+  
+
+**Update mux pod resources**
+
+1. ssh into the logging machine and use the command line.  
+**Note:** You can probably do all of this with the OpenShift UI too - https://logging-machine:8443
+
+2. Run
+
+       # $ oc edit dc logging-mux
+
+Edit the section
+
+      spec:
+        template:
+          spec:
+            containers:
+      ...
+              resources:
+                limits:
+                  cpu: 500m
+                  memory: 2Gi
+
+   For cpu, units are in millicores.  500m means 0.5 cores.
+
+   If you want mux to use more, use 1000m or even 2000m.
```

Comment:
1000m is the limit.  Fluentd/mux is not truly multithreaded.  It will not scale to multiple cores in a single process.
```
If the process is still maxed out at 100% CPU, scale up another pod or more:

oc scale --replicas=2 dc/logging-mux
```